src/retrace-server-reposync

```diff
@@ -5,16 +5,16 @@ import os
 import logging
 import grp
 import pwd
-import rpm
 import shutil
 import sys
 import tempfile
 import urllib2
+import rpm
 import yum
 from retrace import *
+from plugins import *
 
 sys.path.insert(0, "/usr/share/retrace-server")
-from plugins import *
```

Comment:
this was done on purpose. Notice the `sys.path.insert(0, "/usr/share/retrace-server")` part.